README.adoc

```diff
@@ -1,25 +1,27 @@
 
-WARNING: Jenkins is currently (version 2.390) not ready for this plugin to be used to enforce Content-Security-Policy for most resources in production environments.
-Many features, both in core and plugins, will stop working with the default rule set.
-At this time, this plugin is a utility for Jenkins developers, not for Jenkins administrators.
+WARNING: Jenkins is currently (version 2.485) not ready for this plugin to be used to enforce Content Security Policy for all production environments.
+Some features will stop working with the default rule set when _Report Only_ is unchecked.
+See https://issues.jenkins.io/browse/JENKINS-60865[JENKINS-60865] to track plugin compatibility.
 
 == Introduction
 
-This plugin implements Content-Security-Policy protection for the classic Jenkins UI.
+This plugin implements Content Security Policy protection for Jenkins.
 
 == Getting started
 
-Install this plugin to have basic reporting of Content-Security-Policy violations in Jenkins:
+Install this plugin to have basic reporting of Content Security Policy violations in Jenkins:
 A new link _Content Security Policy Reports_ on the _Manage Jenkins_ page allows administrators to review reported policy violations.
 
 Rules can be configured on the _Configure Global Security_ configuration screen.
+By default, Content Security Policy violations are reported but not enforced.
+To enforce Content Security Policy, uncheck _Report Only_ on the _Configure Global Security_ configuration screen.
 
-This plugin serves Content-Security-Policy headers for all HTTP responses, including user-generated content (files in workspaces, archived artifacts, etc.), unless those are served from the https://www.jenkins.io/doc/book/security/user-content/#resource-root-url[Resource Root URL].
-This interacts with the https://www.jenkins.io/doc/book/security/configuring-content-security-policy/[default Content-Security-Policy headers set by Jenkins since 1.641 and LTS 1.625.3 for these resources] as follows:
+This plugin serves Content Security Policy headers for all HTTP responses, including user-generated content (files in workspaces, archived artifacts, etc.), unless those are served from the https://www.jenkins.io/doc/book/security/user-content/#resource-root-url[Resource Root URL].
+This interacts with the https://www.jenkins.io/doc/book/security/configuring-content-security-policy/[default Content Security Policy headers set by Jenkins since 1.641 and LTS 1.625.3 for these resources] as follows:
 
 * If this plugin is configured to only report violations (the default), both enforcing (from Jenkins) and non-enforcing (from this plugin) headers will be set.
-* If this plugin is configured to enforce rules, Jenkins's Content-Security-Policy headers for these resources take precedence over this plugin's.
+* If this plugin is configured to enforce rules, Jenkins's Content Security Policy headers for these resources take precedence over this plugin's.
```

Comment:
This is a weird one, as it's specifically about the header.
```suggestion
* If this plugin is configured to enforce rules, Jenkins's Content-Security-Policy header for these resources takes precedence over this plugin's.
```
to make that clear. The other headers, including X-WebKit-CSP, are set either way, in a conflicting manner, so the existing wording was wrong.